install/audit-logs.tf

```diff
@@ -0,0 +1,95 @@
+resource "aws_s3_bucket" "taskcluster_audit_logs" {
+  bucket = "taskcluster-staging-logs"
```

Comment:
Should this have `audit-` in it too, for consistency?  It doesn't really matter, but if it's easy to change..